internal/runes/runes.go

```diff
@@ -0,0 +1,24 @@
+package runes
+
+import "github.com/samber/lo"
+
+// IndexNotByte is similar with strings.IndexByte but showing the opposite behavior.
+func IndexOfNot(x []rune, c byte) int {
+	i := lo.LastIndexOf(x, ' ')
+
+	if i == -1 && i+1 < len(x) {
```

Comment:
This condition seems rather odd. If `i == -1`, why not just write `&& len(x) > 0` and `return 0`?

Reply:
Resolved in next commit